Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/VmServiceWrapper.java

```diff
@@ -198,6 +198,12 @@ public void addBreakpoint(@NotNull final String isolateId,
                             @NotNull final XLineBreakpoint<XBreakpointProperties> xBreakpoint,
                             @NotNull final VmServiceConsumers.BreakpointConsumerWrapper consumer) {
     final XSourcePosition position = xBreakpoint.getSourcePosition();
+    addBreakpointFromPosition(isolateId, position, consumer);
+  }
+
+  public void addBreakpointFromPosition(@NotNull final String isolateId,
+                                        @NotNull XSourcePosition position,
+                                        @NotNull final VmServiceConsumers.BreakpointConsumerWrapper consumer) {
     if (position == null || position.getFile().getFileType() != DartFileType.INSTANCE) {
```

Comment:
Just to make sure that inspections work in your setup, do you see this yellow code?
![image](https://cloud.githubusercontent.com/assets/4499789/18438233/1e890642-7909-11e6-945b-ab4de3b4557c.png)
Unless there are reasons not to do so try to avoid introducing new warnings. I'll change `@NotNull` to `@Nullable` in `XSourcePosition position` argument annotation to fix yellow code.